README.md

```diff
@@ -18,90 +28,66 @@ One of the most important aspects of collaboration is knowing who else you're wo
 
 ### Where is each user within the application?
 
-Knowing where each user is within an application helps you understand their attentions without always needing to explicitly ask them. For example, seeing that a colleague is currently viewing slide 2 of a slideshow means that you can carry out your own edits to slide 3 without interfering with their work. Displaying the locations of your users can be achieved by highlighting the UI element they have selected, displaying a miniature avatar stack on the slide they are viewing, or showing the live location of their cursors.
+Knowing where each user is within an application helps you understand their intentions without having to explicitly ask them. For example, seeing that a colleague is currently viewing slide 2 of a slideshow means that you can carry out your own edits to slide 3 without interfering with their work. Displaying the locations of your users can be achieved by highlighting the UI element they have selected, displaying a miniature avatar stack on the slide they are viewing, or showing the live location of their cursors.
 
 ### What is everyone doing in the application?
 
 Seeing where users are within an application aids in understanding what they may be working on. It's possible to go one step further though and see what changes they're making to an application. For example, you can display a typing indicator when a colleague is editing a cell in a spreadsheet, or even update the contents of the cell as they type it.
```

Comment:
It's not exactly one step further in this case. I think we need to separate the contextual awareness part here from the actual updates to the app state they are making. So maybe something which says 'everyone should be working on the latest state of the application, so it's crucial that all changes are synced with the backend and immediately reflected in the UI for all connected clients'?

Reply:
I've rewritten this - lemme know if it's clearer.